molo/globalsite/templates/globalsite/country_site.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% load wagtailimages_tags wagtailsettings_tags i18n static %}
+{% get_settings %}
+
+{% block content %}
+
+{% if settings.globalsite.GlobalSiteSettings.description %}
+  <p>{{settings.globalsite.GlobalSiteSettings.description}}</p>
+{% endif %}
+
+
+<h1 class="">{% trans "Choose your country:" %}</h1>
+<div class="">
+  {% for region in regions %}
+    {{region}}
+    {% for country in region.country_sites.all %}
+      <li class="">
+        <a href="{% url 'molo.globalsite:set_country' country.code %}" class="">
+          <div class="">{% if country.flag %}
```

Comment:
I think the if statement needs to be outside the `<div>` and the `class=""` is not necessary.